apps/api/src/app/controllers/__tests__/__snapshots__/fetchViewModelById.e2e.spec.ts.snap

```diff
@@ -10,6 +10,14 @@ Object {
 }
 `;
 
+exports[`GET /entities (fetch view models) When querying for a single View Model by ID GET /photographs/:id when the entity is published when an entity with the id is found should return the expected response 1`] = `
+Object {
+  "id": "0",
+  "imageURL": "https://www.myaudio.com:5555/media?id=cat.png.png",
```

Comment:
Not sure where the .png.png and the .mp3.mp3 are coming from?

Reply:
Fixed the .png one